web/src/main/java/com/bazaarvoice/emodb/web/migrator/LocalMigratorMonitor.java

```diff
@@ -172,10 +171,7 @@ private void refreshMigration(String id)
         }
 
         // Update the set of active migrations
-        if (_activeMigrations.add(id)) {
-            // Schedule a callback to cancel the migration if it goes overrun
-            scheduleOverrunCheck(status);
```

Comment:
First I was thinking about configuring OVERRUN_MIGRATION_TIME in the config file to a week or so and keeping this code, but its a one time thing in each env and we do monitor it closely, so I think I agree with wiping this code out.